docs/markdown/ios/card/main_component.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Card
+description: The Card is a semi-modular container used to house a Card.Header and a variant of the available preview blocks. Every Card must have at least a title and a variant of a preview block.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/fb/df/a0/fbdfa01750ca4951fde9da763052c9de.png" alt="Example of a Card container with an area to show where images are placed." noPadding color="background-elevation-accent"/>
+
+
+## Usage guidelines
+
+<TwoCol>
+<Group>
+<Do title="When to use"/>
+- Previewing information about recommended content (ex: pin feed, boards, profiles, topics, etc) that also serves as an entry point to a destination containing the content
```

Comment:
nit: we're always supposed to capitalize "Pin"

Reply:
good catch! will update